.github/workflows/compatibility.yaml

```diff
@@ -0,0 +1,126 @@
+name: Compatibility of
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '21 5,17 * * *'
+
+jobs:
+  latest:
+    runs-on: ubuntu-latest
+        
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Rename files
+        working-directory: .
+        run: |
+          cp template.env .env
+          cp local_nb_nodes.template.json local_nb_nodes.json
+          sed -i 's/latest/nightly/g' .env
```

Comment:
I believe this is replacing `latest` with `nightly` so we're running nightly here too.

Reply:
oh wow, thanks for catching that!